docs/sparkr.md

```diff
@@ -210,6 +210,43 @@ head(df)
 {% endhighlight %}
 </div>
 
+### Model Formulae
+
+SparkR allows the fitting of generalized linear models over DataFrames using the glm() function. Under the hood, SparkR uses MLlib to train a model of the specified family. In Spark 1.5, we support a subset of the available R formula operators for model fitting, including '~', '.', '+', and '-'. The example below shows the use of R formulae with SparkR.
```

Comment:
Actually it'll be better to talk about the families and link functions supported here before the formula. Also we could reword the example sentence to say `the example below shows the use of building a gaussian GLM model using SparkR`.